api-gateway/src/main/java/org/swasth/apigateway/models/BaseRequest.java

```diff
@@ -92,9 +99,19 @@ protected void validateCondition(Boolean condition, ErrorCodes errorcode, String
     }
 
     protected void validateParticipant(Map<String,Object> details, ErrorCodes code, String participant) throws ClientException {
+        String participantCode = "1" + (String) details.get("participant_code");
```

Comment:
why are we appending the code with "1"?

Reply:
in registry response we get participant code without "1".